### ISSUE TITLE
[Keyboard Navigation-Help-Getting Started]Focus order is not logical on the ‘Getting Started’ Page.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2548]
**User Experience:**
If focus moves here and there and does not follow proper order then it would be difficult to access any Page for a Screen reader dependent users.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Help control lying in the header section on the page and select it.
4. Navigate to 'Getting started' control and select it.
5. Verify that focus order on the ‘Getting Started’ Page.

**Actual Result:-**
Focus order is not logical on the ‘Getting Started’ Page as focus first moves to previous page elements then it moves to OK button.

**Expected Result:-**
Focus order should be logical on the ‘Getting Started’ Page as focus first does not moves to previous page elements it directly moves to OK button.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Suggested Fix:**
tab index property is applicable to all visual elements on screen.  
<a href="/articles/" tab index="2">Articles</a> 
<a href="/resources/" tab index="3">Resources</a> 
 
Tips : 
'tab index' should only be used in cases where: 
•The default tab order is not ideal, AND 
•The tab order cannot be changed by rearranging items in the content and/or by altering the style sheet to reflect the best visual arrangement. 
 
**Key References for Fix:** 
http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20160317/G59 
https://www.w3.org/WAI/GL/wiki/Creating_Logical_Tab_Order_with_the_Tabindex_Attribute 

**Please refer the Screen shot:**

[MAS2.4.3_Help_Getting started.zip](https://github.com/Microsoft/pxt/files/1179768/MAS2.4.3_Help_Getting.started.zip)

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.microbit.org/acc and bug is fixed .Hence closed the bug by added closed tag.
